src/openzaak/components/zaken/api/filters.py

```diff
@@ -100,6 +183,19 @@ class ZaakFilter(FilterSet):
             help_text=get_help_text("zaken.OrganisatorischeEenheid", "identificatie"),
         )
     )
+    rol__machtiging = filters.ChoiceFilter(
+        field_name="rol__indicatie_machtiging",
+        method=machtiging_filter,
+        help_text=MACHTIGING_HELP_TEXT,
+        choices=MachtigingChoices.choices,
+    )
+    rol__machtiging__loa = MaxLoAFilter(
+        field_name="rol__authenticatie_context__level_of_assurance",
+        help_text=mark_experimental(
+            "Zaken met een `rol.authenticatieContext.levelOfAssurance` die beperkter is dan de "
```

Comment:
does `beperkter` here mean a lower level of assurance? If that is the case, the text implies that cases with a rol with the specified `levelOfAssurance` will not be shown

Reply:
it should be the current level and everything that lower. So it's similar to `vertrouwelijkheidaanduiding` field. Could you help to write a correct help text in Dutch please?

Reply:
Maybe something like `Alleen Zaken met een vertrouwelijkheidaanduiding die lager dan of gelijk is aan de aangegeven aanduiding worden teruggeven als resultaten.`